.gitattributes

```diff
@@ -4,3 +4,11 @@ EXAConf       text eol=lf
 
 poetry.lock linguist-generated=true
 exasol_integration_test_docker_environment/docker_db_config/**/* linguist-generated=true
+.github/workflows/cd.yml linguist-generated=true
+.github/workflows/check-release-tag.yml linguist-generated=true
+.github/workflows/checks.yml linguist-generated=true
+.github/workflows/ci.yml linguist-generated=true
+.github/workflows/gh-pages.yml linguist-generated=true
+.github/workflows/merge-gate.yml linguist-generated=true
+.github/workflows/pr-merge.yml linguist-generated=true
+.github/workflows/report.yml linguist-generated=true
```

Comment:
AFAIK the workflows initially are generated by PTB, yes. 
But AFAIK there is no automation to keep the workflows up-to-date.
Hence, we cannot be sure that workflows will remain _generated_ in future. 

I therefore propose to **not mark** the workflows as "generated" until PTB provides an automation [similar to project-keeper](https://github.com/exasol/project-keeper/blob/main/doc/user_guide/user_guide.md#customize-workflow-steps) allowing to use generated workflows but apply small adjustments, though.

Reply:
There is

Reply:
To me, it still looks different. There are commands `install`, `diff`, and `update` but I can't see "customize" or similar.

Reply:
There is update and diff for the customized onecr. However, it is true, that it is not automatic, because the workflows are often customized. We probably need to modify the workflows in a way, that they call defined extension workflows. So, maybe adding them not automatically to generated is at least a good idea

Reply:
I agree with @tkilias  until this is fully automated, these workflows should be treated as manually written.